### DISCUSSION TITLE
Updating CSLA from v5.4.0 to v8.0.0

### DISCUSSION THREAD

**Discussion by Guidemarcus** [MarimerLLC/csla#4179] [Questions] [answered]
Hi!

I am updating the CSLA package of my project from 5.4.0 to 8.0.0. I am having trouble finding clear migration steps that could help me with my task. Does this kind of documentation exist?

In my project, we de not use CSLA too heavily. We were overriding BusinessBase and using some DataPortal functions. I was able to find some stuff online with the help of chatgpt, but I find that I am still missing some context to be comfortable updating my project.

**Comment by rockfordlhotka** [comment, marked as answer]
Have you looked at the upgrade docs here?

https://github.com/MarimerLLC/csla/tree/main/docs

**Reply by Guidemarcus** [reply]
yes thank you. I was not able to find the doc about the updates
